torch/_inductor/codegen/common.py

```diff
@@ -2025,27 +2027,81 @@ def sort(
             @staticmethod
             def bucketize(
                 values: CSEVariable,
-                offsets_name: str,
-                offsets_size: sympy.Expr,
+                boundaries: Tuple[str, sympy.Expr, sympy.Expr, sympy.Expr],
```

Comment:
just curious, is there a reason to provide this as a tuple instead of 4 separate args?

Reply:
@davidberard98 I agonized over this interface for a while. Passing it as 4 separate args was the simplest approach, but also made the function signature really bloated. It doesn't technically make any difference, but I often find that a shorter function signature is a surprisingly good aid to developer comprehension, by appropriately grouping things together as needed.

As another way of justifying this... `boundaries` here represents a tensor, and it makes sense to me to pass pointer/size/strides as a single, tensor-lite object.

Reply:
Sure, makes sense